src/rickroll.js

```diff
@@ -0,0 +1,10 @@
+const Discord = require('discord.js')
+const LINK = 'https://i.pinimg.com/originals/78/3d/4f/783d4f34998ae3b3b576e400db0d81df.gif'
+
+async function rr(message) {
+  if (message.author.bot) return
+
+  message.channel.send(LINK)
+}
+
+module.exports = rickroll
```

Comment:
The export is referring to a function named `rickroll` while the function is called `rr`

Reply:
Ok sorry I was rushed because I did it at school I have multiple packages but it seems I dont know how to do things when I am rushed by time thanks for taking the time to look at the PR though